# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/script/moz.build
# Commit: a48aff5f2555
# Full Hash: a48aff5f2555af61a9bc1ee0d77f6058f47d0501
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 09:08:22
# Regressor Bug: 1742437
# File Overlap Count: 2
# Description:
#   Bug 1742437 - Move module loader files into a central folder; r=jonco
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D137532
# ==============================================================================

diff -r 7e54a223001d -r a48aff5f2555 dom/script/moz.build
--- a/dom/script/moz.build	Thu Feb 17 15:57:51 2022 +0000
+++ b/dom/script/moz.build	Thu Feb 17 15:57:52 2022 +0000
@@ -19,37 +19,29 @@
 
 EXPORTS.mozilla.dom += [
     "AutoEntryScript.h",
-    "LoadedScript.h",
     "ModuleLoader.h",
-    "ModuleLoaderBase.h",
-    "ModuleMapKey.h",
     "ScriptDecoding.h",
     "ScriptElement.h",
-    "ScriptKind.h",
     "ScriptLoadContext.h",
     "ScriptLoader.h",
-    "ScriptLoadRequest.h",
     "ScriptSettings.h",
+    "ScriptTrace.h",
 ]
 
 UNIFIED_SOURCES += [
     "AutoEntryScript.cpp",
-    "LoadedScript.cpp",
     "ModuleLoader.cpp",
-    "ModuleLoaderBase.cpp",
-    "ModuleLoadRequest.cpp",
-    "ModuleMapKey.cpp",
     "nsIScriptElement.cpp",
     "ScriptElement.cpp",
     "ScriptLoadContext.cpp",
     "ScriptLoader.cpp",
     "ScriptLoadHandler.cpp",
-    "ScriptLoadRequest.cpp",
     "ScriptSettings.cpp",
 ]
 
 LOCAL_INCLUDES += [
     "/dom/base",
+    "/js/loader",
 ]
 
 include("/ipc/chromium/chromium-config.mozbuild")